modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -1,76 +1,86 @@
 [id='proc-customize-rhdh-sidebar-menuitems_{context}']
 = Customizing the sidebar menu items for your {product-short} instance
 
-The sidebar menu in {product} consists of two main parts:
+The sidebar menu in {product} consists of two main parts that you can configure:
 
-* *Main menu items*: These items are the static menu items that form the core navigation structure of sidebar. These menu items remain consistent and are predefined.
+Dynamic plugin menu items:: Your preferences and your active plugins define dynamically one part of the sidebar menu.
+Main menu items:: The core navigation structure of sidebar is static.
 
-* *Dynamic plugin menu items*: These items are displayed beneath the main menu and can be customized based on the plugins installed. The main menu items section is dynamic and can change based on your preferences and installed plugins.
-
-.Procedure
-
-. Customize the main menu items using the following steps:
+.Default static menu item configuration
 +
---
-.. Open the `app-config-rhdh.yaml` file.
-.. To customize the order and parent-child relationships for the main menu items, use the `dynamicPlugins.frontend.default.main-menu-items.menuItems` field.
-.. For dynamic plugin menu items, use the `dynamicPlugins.frontend.<package_name>.menuItems` field.
-
-.Example `app-config-rhdh.yaml` file
 [source,yaml]
```

Comment:
This code block is not YAML. `export const` denotes the creation of a variable in some programing language, probably a JavaScript variant.